package.json

```diff
@@ -120,7 +120,6 @@
     "font-awesome": "4.7.0",
     "fp-ts": "^2.8.5",
     "has-ansi": "^4.0.0",
-    "he": "^1.2.0",
```

Comment:
did you run `npm install`? there should be changes to `package-lock` as well right?